speculos/mcu/display.py

```diff
@@ -81,11 +93,33 @@ def draw_point(self, x, y, color):
         self.pixels[(x, y)] = color
 
     def screenshot_update_pixels(self):
+        # Someone tampered our self.pixels content.. update the screenshot to reflect it
+        # NANO only
         self.screenshot.update(self.pixels)
 
     def take_screenshot(self):
         return self.screenshot.get_image()
 
+    def publish_screenshot(self):
```

Comment:
I would rather call this "save_screenshot" rather than publish as it sounds like it is sent somewhere